docs/reference/migration/migrate_7_6.asciidoc

```diff
@@ -14,12 +14,44 @@ See also <<release-highlights>> and <<es-release-notes>>.
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+[discrete]
+[[breaking_76_security_changes]]
+=== Security changes
+
+[discrete]
+==== {es} authentication API key privileges
+
+If you use the {es} API key service to generate API keys, the behavior of new
+API keys is impacted by the fix for
+https://www.elastic.co/community/security[CVE-2020-7009].
+
+When you make a request to create API keys, you can specify an expiration and
+permissions for the API key. Previously, if you used an API key to create
+another API key (sometimes called a _derived key_), by default it had no
```

Comment:
Should we be more explicity about the previous behaviour? that is the derived key had no privilege regardless of the role descriptors specified at creation time